hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -22,7 +22,7 @@
   sw_checklist:       "/sw/device/lib/dif/dif_keymgr",
   revisions: [
     {
-      version:            "1.0.0",
+      version:            "2.0.0",
```

Comment:
Increasing major version due to multiple backwards-incompatible SW-visible changes